_pages/changelog.md

```diff
@@ -5,7 +5,17 @@ include_in_header: true
 ---
 # Change Log
 
-## Release 2.4.2 `latest`
+## Release 3.0.0 `latest`
+**Jan 25, 2024**
+
+- Added support for multiple MPD server connections
+- Automatic discovery of local MPD instances using Bonjour/Zeroconf protocol
+- Ability to define multiple local playback stream URLs (with various bitrate and audio quality)
```

Comment:
```suggestion
- Ability to define multiple local playback stream URLs (with various buffer sizes)
```

Reply:
Yes, but buffer size is only a tech detail of our configuration model. What is important for users is that they can define different MPD streaming outputs with various bitrate/quality. IMO better for marketing and more communicative for users outside of our detailed config context.